packages/random/mt.pony

```diff
@@ -1,26 +1,28 @@
 class MT is Random
   """
-  A Mersenne Twister. This is a non-cryptographic random number generator.
+  A Mersenne Twister. This is a non-cryptographic random number generator. This
+  should only be used for legacy applications that require a Mersenne Twister,
+  otherwise use Rand.
   """
   embed _state: Array[U64]
   var _index: USize
 
-  new create(seed: U64 = 5489) =>
+  new create(x: U64 = 5489, y: U64 = 0) =>
     """
     Create with the specified seed. Returned values are deterministic for a
```

Comment:
I don't thinks this method comment is helpful at this point. There's two values, but it says "with the specificied seed". This should be updated. 

I think it makes sense to update the class comment with a usage example as well before merging this.